File: src/tool/spring-boot-generator/output/authority/db/userInfo.js
let userInfo = {
	name: "userInfo",
	description: "user info",
	columns: {
		account: {
			name: "account",
			description: "account",
			isPk: true,
			nullable: false,
			type: {
				name: "varchar",
				length: null
			}
		},
		password: {
			name: "password",
			description: "password",
			nullable: false,
			type: {
				name: "varchar",
				length: null
			}
		},
		createTime: {
			name: "createTime",
			description: "create time",
			nullable: false,
			type: {
				name: "timestamp",
				length: null
			}
		},
		createUser: {
			name: "createUser",
			description: "create user",
			nullable: false,
			type: {
				name: "varchar",
				length: null
			}
		}
	}
};

let columnsArray = [
	"account",
	"password",
	"createTime",
	"createUser"
];

function create() {
  return {};
}

userInfo.columnsArray=columnsArray;
userInfo.create=create;
module.exports=userInfo;

File: src/tool/spring-boot-generator/output/authority/authority/src/main/java/com/authority/pojo/entity/UserInfo.java
package com.authority.pojo.entity;

import io.swagger.annotations.ApiModel;
import io.swagger.annotations.ApiModelProperty;
import lombok.Getter;
import lombok.Setter;

import java.util.Date;



@ApiModel("user info")
@Getter
@Setter
public class UserInfo  {

    @ApiModelProperty("account")
    private String account;

    @ApiModelProperty("password")
    private String password;

    @ApiModelProperty("create time")
    private Date createTime;

    @ApiModelProperty("create user")
    private String createUser;

}